### PR TITLE
NAS-125942 / 24.04 / Add unique id to sentry tags and new tickets payload

### PR DESCRIPTION
**Pull request by denysbutenko**
Add host id and session id to sentry events and new tickets payload.

### CODE REVIEW

File: src/app/services/system-general.service.ts [hunk 1]
```diff
@@ -48,12 +52,18 @@ export class SystemGeneralService {
...
-    ]).subscribe(([isStable, sysInfo]) => {
+      this.store$.select(selectSystemHostId).pipe(filter(Boolean)),
+      this.sessionId$,
+    ]).subscribe(([isStable, sysInfo, hostId, sessionId]) => {
       if (!isStable) {
         Sentry.init({
           dsn: environment.sentryPublicDsn,
           release: sysInfo.version,
         });
+        Sentry.configureScope((scope) => {
```
Comment (undsoft) [inline src/app/services/system-general.service.ts:63]: I think it's time to create a new service.

Comment (denysbutenko) [reply src/app/services/system-general.service.ts:63]: Good idea 👍🏻

**Review by AlexKarpov98 — APPROVED** [review]
Looks good

**Review by undsoft — APPROVED** [review]
Looks good, I was able to find my errors in Sentry after reporting a bug.
Don't forget to change `if (!isStable) {` to correct condition in the service before merging.

**Comment by undsoft** [comment]
Also it's worth looking into what conditions we have for enabling error reporting. I think we should still report errors on nightlies to Sentry, but we shouldn't report them if it's a dev machine (localhost).

**Comment by denysbutenko** [comment]
> Looks good, I was able to find my errors in Sentry after reporting a bug. Don't forget to change `if (!isStable) {` to correct condition in the service before merging.

What is the correct condition? It wasn't changed in this PR.

> Also it's worth looking into what conditions we have for enabling error reporting. I think we should still report errors on nightlies to Sentry, but we shouldn't report them if it's a dev machine (localhost).

Before there was an additional flag to enable reporting, but it was removed from the middleware side. I agree it needs to be revisited. @undsoft would you mind creating a new ticket for that and assign to me?

**Comment by undsoft** [comment]
Well, I would imagine that it's when the system is stable that we mostly want to submit issues. I'll describe this in the new ticket.

**Pull request merged**